src/aics-image-viewer/shared/constants.ts

```diff
@@ -110,13 +124,7 @@ export const getDefaultViewerState = (): ViewerState => ({
   region: { x: [0, 1], y: [0, 1], z: [0, 1] },
   slice: { x: 0.5, y: 0.5, z: 0.5 },
   time: 0,
-  cameraState: {
-    position: [0, 0, 5],
-    target: [0, 0, 0],
-    up: [0, 1, 0],
-    fov: 20,
-    orthoScale: 0.5,
-  },
+  cameraState: undefined,
```

Comment:
It occurs to me also that a named constant like `USE_CAMERA_FOR_VIEW_MODE=undefined` could help with readability.  Or else use a separate boolean variable flag rather than overloading the camera default setting type.

Reply:
Added a named constant here.